src/decorator/live-decorator.ts

```diff
@@ -1,7 +1,7 @@
 import { KalturaLivePlugin } from "../kaltura-live-plugin";
 import { getContribLogger } from "@playkit-js-contrib/common";
 
-const HTTP_ERROR = 1002;
+const HTTP_ERRORS = [1002, 1003];
```

Comment:
@esakal  - I feel that we need to give a more robust logic here than to hunt these cases error by error. 
We do have a severity and category and this is the delta between Chrome and Safari 

Chrome 
category: 1
code: 1002
data: {name: "levelLoadError", url: undefined, responseCode: 0}
severity: 2


Safari 
category: 1
code: 1003
data: "The player exceeded max buffer time of 30000 ms. No progress has been done during this time."
severity: 2